tests/jax/test_distributed_fused_attn.py

```diff
@@ -355,6 +354,10 @@ def test_context_parallel_allgather_attn(
             CPStrategy.ALL_GATHER,
         )
 
+    @pytest.mark.parametrize(
+        "use_scan",
+        [pytest.param(False, id="NO_SCAN"), pytest.param(True, id="USE_SCAN")],
+    )
```

Comment:
https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/jax/cpp_extensions/attention.py#L1605-L1608
We need change here to `return use_scan` directly, otherwise it will always fall through to no scan

Reply:
`--xla_experimental_ignore_channel_id` was a WAR, and the latest XLA should address that.